react/table/src/auth/keycloak/useKeycloak.js

```diff
@@ -0,0 +1,4 @@
+const useKeycloak =
```

Comment:
after the introduction of hooks in react, better avoiding using use as a prefix, because now it seems the name of a hook :)